src/org/opendatakit/briefcase/ui/SettingsPanel.java

```diff
@@ -38,38 +56,102 @@ public SettingsPanel(MainBriefcaseWindow parentWindow) {
+            					.addGap(4)
+            					.addComponent(txtHost, GroupLayout.DEFAULT_SIZE, 88, Short.MAX_VALUE)
+            					.addPreferredGap(ComponentPlacement.RELATED)
+            					.addComponent(lblPort)
+            					.addGap(3)
+            					.addComponent(txtPort, GroupLayout.DEFAULT_SIZE, 39, Short.MAX_VALUE)
+            					.addPreferredGap(ComponentPlacement.RELATED, 14, Short.MAX_VALUE)
+            					.addComponent(btnSave, GroupLayout.DEFAULT_SIZE, GroupLayout.DEFAULT_SIZE, Short.MAX_VALUE))
+            				.addGroup(Alignment.TRAILING, groupLayout.createSequentialGroup()
+            					.addComponent(lblBriefcaseDirectory)
+            					.addGap(18)
+            					.addComponent(txtBriefcaseDir, GroupLayout.DEFAULT_SIZE, 61, Short.MAX_VALUE)
+            					.addGap(18)
+            					.addComponent(btnChoose))
+            				.addComponent(lblGeneralSettings)
+            				.addComponent(lblProxySettings, GroupLayout.PREFERRED_SIZE, 132, GroupLayout.PREFERRED_SIZE))
+            			.addContainerGap())
+            );
+            groupLayout.setVerticalGroup(
+            	groupLayout.createParallelGroup(Alignment.LEADING)
+            		.addGroup(groupLayout.createSequentialGroup()
+            			.addGap(18)
+            			.addComponent(lblGeneralSettings)
+            			.addPreferredGap(ComponentPlacement.RELATED)
+            			.addGroup(groupLayout.createParallelGroup(Alignment.BASELINE)
+            				.addComponent(txtBriefcaseDir, GroupLayout.PREFERRED_SIZE, GroupLayout.DEFAULT_SIZE, GroupLayout.PREFERRED_SIZE)
+            				.addComponent(btnChoose)
+            				.addComponent(lblBriefcaseDirectory))
+            			.addGap(14)
+            			.addComponent(lblProxySettings, GroupLayout.PREFERRED_SIZE, 17, GroupLayout.PREFERRED_SIZE)
+            			.addPreferredGap(ComponentPlacement.RELATED)
+            			.addGroup(groupLayout.createParallelGroup(Alignment.BASELINE)
+            				.addComponent(lblSchema)
+            				.addComponent(schemaComboBox, GroupLayout.PREFERRED_SIZE, GroupLayout.DEFAULT_SIZE, GroupLayout.PREFERRED_SIZE)
+            				.addComponent(txtHost, GroupLayout.PREFERRED_SIZE, GroupLayout.DEFAULT_SIZE, GroupLayout.PREFERRED_SIZE)
+            				.addComponent(lblHost)
+            				.addComponent(txtPort, GroupLayout.PREFERRED_SIZE, GroupLayout.DEFAULT_SIZE, GroupLayout.PREFERRED_SIZE)
+            				.addComponent(lblPort)
+            				.addComponent(btnSave))
+            			.addGap(314))
+            );
 
         setLayout(groupLayout);
 
         navOrder.add(lblBriefcaseDirectory);
         navOrder.add(txtBriefcaseDir);
         navOrder.add(btnChoose);
+        
+        setCurrentProxySettings();
     }
+    
+    private void setCurrentProxySettings() {
+      String currentProxyType = BriefcasePreferences.getBriefCaseProxyType();
+	  if (currentProxyType.equals(ProxyConnection.ProxyType.NO_PROXY.toString())){
```

Comment:
null check here instead for no proxy, return immediately after setting the schemaComboBox?